app/build.gradle

```diff
@@ -146,7 +146,7 @@ android {
 dependencies {
   implementation project(path: ':router')
 
-  testImplementation "junit:junit:$versions.junit"
+  implementation "junit:junit:$versions.junit"
```

Comment:
Should we change this to `debugImplementation` instead?

Reply:
Done